src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/entity/ActionPlan.kt

```diff
@@ -0,0 +1,38 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity
+
+import java.time.OffsetDateTime
+import java.util.UUID
+import javax.persistence.CollectionTable
+import javax.persistence.ElementCollection
+import javax.persistence.Entity
+import javax.persistence.FetchType
+import javax.persistence.Id
+import javax.persistence.JoinColumn
+import javax.persistence.ManyToOne
+import javax.persistence.Table
+import javax.validation.constraints.NotNull
+
+@Entity
+@Table(name = "action_plan")
+data class ActionPlan(
+
+  // Attributes
+  var numberOfSessions: Int? = null,
+
+  // Activities
+  @ElementCollection
+  @CollectionTable(name = "action_plan_activity", joinColumns = [JoinColumn(name = "action_plan_id")])
+  @NotNull var activities: List<ActionPlanActivity>,
```

Comment:
can we default this to `emptyList()` and make it `val`?

Reply:
Will do